pkg/cache/clusterqueue.go

```diff
@@ -263,15 +267,18 @@ func (c *ClusterQueue) inactiveReason() (string, string) {
 	case terminating:
 		return "Terminating", "Can't admit new workloads; clusterQueue is terminating"
 	case pending:
-		switch {
-		case c.hasMissingFlavors && c.hasMissingOrInactiveAdmissionChecks:
-			return "FlavorNotFoundAndCheckNotFoundOrInactive", "Can't admit new workloads; some resourceFlavors are not found and admissionChecks are not found or inactive"
-		case c.hasMissingFlavors:
-			return "FlavorNotFound", "Can't admit new workloads; some resourceFlavors are not found"
-		default:
-			return "CheckNotFoundOrInactive", "Can't admit new workloads; some admissionChecks are not found or inactive"
-
+		reasons := make([]string, 0, 3)
+		if c.isStopped {
+			reasons = append(reasons, "Stopped")
+		}
+		if c.hasMissingFlavors {
+			reasons = append(reasons, "FlavorNotFound")
 		}
+		if c.hasMissingOrInactiveAdmissionChecks {
+			reasons = append(reasons, "CheckNotFoundOrInactive")
+		}
+
+		return strings.Join(reasons, "_"), strings.Join([]string{"Can't admit new workloads:", strings.Join(reasons, ", ")}, " ")
```

Comment:
Don't concatenate the reasons. It will make it harder for users to write controllers or alerts based on the reason.

Let's just return one reason. If there are other problems, once they solve the first problem, they will see the next.